content/blog/adoptopenjdk-8u262-1108-and-1402-available/index.md

```diff
@@ -0,0 +1,64 @@
+
+* [List of security fixes and other fixes in OpenJDK 14.0.2](https://bugs.openjdk.java.net/browse/JDK-8248956?jql=project%20%3D%20JDK%20AND%20fixVersion%20%3D%2014.0.2%20ORDER%20BY%20created%20DESC)
+
+* [List of changes in Eclipse OpenJ9 0.21.0](https://github.com/eclipse/openj9/blob/master/doc/release-notes/0.21/0.21.md)
+
+## New and Noteworthy
+
+### Java Flight Recorder Available in 8u262
+
+Java Flight Recorder (JFR) is a tool for collecting diagnostics of a running Java application. This diagnostic information can, for example, be analyzed with [JDK Mission Control](http://jdk.java.net/jmc/). JFR was previously only available in Oracle JDK 8, and OpenJDK 11 or later. With OpenJDK 8u262, is now also part of OpenJDK 8 and can be used with AdoptOpenJDK without usage restrictions. Flight recording can be enabled on the command line like this:
+
+```
+$ java -XX:StartFlightRecording:filename=recording.jfr,duration=60s ... 
+```
+
+It is also possible to enable flight recording for an already running JVM using `jcmd`:
+
+```
+$ jcmd <pid> JFR.start
+$ jcmd <pid> JFR.dump filename=recording.jfr
+```
+
+Run `jcmd` to get a list of available PIDs. The resulting recording can be analyzed using various tools. A popular choice is JDK Mission Control (JMC) [that is also available from AdoptOpenJDK](https://adoptopenjdk.net/jmc.html).
+
+### Auto-Upgrading Homebrew Casks for JRE, OpenJ9 
+
+Users of AdoptOpenJDK can always get the latest and greatest version of the JDK with HotSpot by installing the `adoptopenjdk` cask:
+
+```
+$ brew cask install adoptopenjdk
+```
+
+We have added additional casks that automatically upgrade to the latest and greatest version of the JRE and OpenJ9, too. If you haven't already, you need to first activate the AdoptOpenJDK tap: 
+
+```
+$ brew tap AdoptOpenJDK/openjdk
+```
+
+You can then choose between these auto-upgrading casks and install them with `brew cask install <name>`: 
+
+* adoptopenjdk
+* adoptopenjdk-jre
+* adoptopenjdk-openj9
+* adoptopenjdk-openj9-jre
+
+## A Note on Binaries for 32-bit ARM
+
+Because we're short on machines with 32-bit ARM CPUs and those we have are also incredibly slow, our testing capacity is minimal. As a result, we haven't caught a bug in the last release that caused certain SSL connections to fail. This has been fixed in the meantime. We are already looking into procuring additional capacity that should make it possible to re-enable our entire test suite and deliver binaries that match our quality criteria.
+
+Another longstanding problem is that AdoptOpenJDK 8 on 32-bit ARM only supports interpreted mode (Zero VM). We would very much like to fix that. Hence, we're looking for volunteers who help us build AdoptOpenJDK 8 with a proper client VM that is already available on AdoptOpenJDK 11 and higher.
```

Comment:
@sxa is this correct? I thought that we were building with a JIT on arm32 now

Reply:
```
jdk8u262-b09/bin/java -version
openjdk version "1.8.0_262"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_262-202007131722-b09)
OpenJDK Zero VM (AdoptOpenJDK)(build 25.262-b09, interpreted mode)
```

Reply:
It's the last nightly build, however.

Reply:
yes you're right - sorry we reverted (https://github.com/AdoptOpenJDK/openjdk-build/commit/4937941d0d2285a191d35824b43a982d76e08a29)

I'll kick off a zero arm32 build now then